scss/components/table.scss

```diff
@@ -166,19 +181,18 @@ $block: #{$fd-namespace}-table;
         }
     }
 
-    th.fd-table__context-menu {
+    .#{$block}__header .#{$block}__cell.fd-table__context-menu {
         &:hover {
             @include fd-var-color("background-color", $fd-table-sort-column-header-background-color, --fd-color-neutral-2);
             cursor: pointer;
         }
     }
 
-    &__fixed-col {
+    &__fixed-cell {
+        left: 0;
+        max-width: 200px;
```

Comment:
When using a fixed column, why does the column have to be 200 pixels wide?  I see lots of hard-coded references to 200px and although it works, it seems very limiting.  That seems like a detail for the implementation library to handle.

A little out of scope for this PR, but maybe we come back to this in a separate issue?

Reply:
I can remove this, and add back the inline styling in example.  With docs instruction, it looks like there is direction for the consumer to set width here.

Reply:
Separate story opened: https://github.com/SAP/fundamental-styles/issues/292